README.md

```diff
@@ -16,84 +16,83 @@ We should be as specific and thorough as possible when defining our style, testi
 
 #### 1. Names of internal functions in a library should not have an underscore prefix.
 
-The style guide states
-
-> Underscore Prefix for Non-external Functions and Variables
-
-One of the motivations for this rule is that it is a helpful visual clue.
+YES:
```

Comment:
I found it more readable to put the "YES" examples before the "NO" examples - also matches solidity's style guide